cmd/lint.go

```diff
@@ -312,6 +316,23 @@ func lint(args []string, params *lintCommandParams) (report.Report, error) {
 		m.Timer(regalmetrics.RegalConfigParse).Stop()
 	}
 
+	go func() {
+		dir := ""
+		if regalDir != nil {
+			dir = regalDir.Name()
```

Comment:
I'm a little weary about having `regal lint` change the worktree for users. They can .gitignore it of course, but it's an extra annoyance. I wonder if we can use something like `~/.config/regal` as the default instead? We'd have to create the directory though, and fall back gracefully if we can't (perhaps by _then_ trying the `.regal` in the project directory?).

Reply:
added in https://github.com/StyraInc/regal/pull/824/commits/72dcabf6d09415a7adee1bd7759cb4f099578222